data/scripts/creaturescripts/customs/freequests.lua

```diff
@@ -1,6 +1,8 @@
 local stage = configManager.getNumber(configKeys.FREE_QUEST_STAGE)
```

Comment:
I think you need to change the stage in config.lua, for those who have the server online and are going to update